src/test/Makefile

```diff
@@ -438,11 +438,19 @@ COMMON_FLAGS = \
 	-O0 \
 	-DUNIT_TEST \
 	-isystem $(GTEST_DIR)/inc \
-	-MMD -MP
+	-MMD -MP \
+	-Wno-c99-extensions \
+	-Wno-reorder
 
 CC_VERSION = $(shell $(CC) -dumpversion)
 CXX_VERSION = $(shell $(CXX) -dumpversion)
 
+# Betaflight supports clang 10. Revisit when new version arrive.
+CC_SUPPORTED_VERSION := $(shell expr `clang -dumpversion | cut -f1 -d.` \>= 10)
+ifeq "$(CC_SUPPORTED_VERSION)" "1"
+    $(error Betaflight requires clang 10.0 or below)
```

Comment:
Probably add here: "On most systems the correct compiler will be available as 'clang-10' and 'clang++-10, but Betaflight invokes 'clang' and 'clang++' to accommodate some operating systems that do not have proper clang compilers available. Please modify your system so that an appropriate version of clang is invoked as 'clang' and 'clang++'."

Reply:
Replaced error message.